client/cli/src/params/import_params.rs

```diff
@@ -52,11 +52,11 @@ pub struct ImportParams {
 
 	#[allow(missing_docs)]
 	#[structopt(flatten)]
-	pub execution_strategies: ExecutionStrategies,
+	pub execution_strategies: ExecutionStrategiesParams,
 
 	/// Limit the memory the database cache can use.
-	#[structopt(long = "db-cache", value_name = "MiB", default_value = "128")]
-	pub database_cache_size: u32,
+	#[structopt(long = "db-cache", value_name = "MiB")]
+	pub database_cache_size: Option<usize>,
```

Comment:
as said before, we don't want to pass `None` to rocksdb but our own default. I think leaving `default_value` in for documentation purposes (and to show it in the cli help ... does this actually?), this is better than it being optional.

Reply:
Unfortunately it makes things less flexible because a binary cannot override the default value anymore. That is why the default value is now in config.rs.

Though maybe for this particular case it is fine if we keep it in the definition... Let me know what you prefer.

(Note: it does show the default value in the --help if we keep it in the definition)

Reply:
Done as suggested in DM